Microsoft.AVS.Management/Microsoft.AVS.Management.psm1

```diff
@@ -2628,5 +2628,57 @@ Function New-AVSStoragePolicy {
     }
 }
 
+<#
+    .Synopsis
+        This allows the customer to change DRS from the default setting to one step more conservative.
+    .PARAMETER Drs
+        The DRS setting to apply to the cluster.  3 is the default setting, 4 is one step more conservative (meaning less agressive in moving VMs).
+    .EXAMPLE
+        Set-CustomDRS -Drs 4
+        Set-CustomDRS -Drs 3 # This returns it to the default setting
+#>
+function Set-CustomDRS {
 
```

Comment:
Maybe it should take cluster as parameter?

Reply:
I went back and forth about that. I'll add it.